modules/must-gather-storage-limit.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="must-gather-storage-limit_{context}"]
+= Changing the must-gather storage limit
+
+When using the `oc adm must-gather` command to collect data the default maximum storage for the information is 30% of the storage capacity of the container. Once that limit is reached the container is killed and the gathering process stops. You will need either a container with more storage capacity or to adjust the maximum volume percentage before running the must-gather command again. Information already gathered will be lost.
```

Comment:
> Information already gathered will be lost.

Actually, we only stop `gather` container to prevent the size getting bigger more. On the other hand, `copy` container will stay alive which means that gathered data will not be lost and downloaded to user. Just since, gather process is no longer working, this gathered data is partial.

Reply:
Thank you! I'll change it.

Reply:
@ardaguclu How does it sound now?

Reply:
Sounds good to me. Thanks @cbippley